### PR TITLE
Show statuses on the first page

### PR DESCRIPTION
**Pull request by geramirez**

### CODE REVIEW

File: client/src/App.scss [hunk 1]
```diff
@@ -12,3 +12,11 @@ html {
     margin: 0.5em 0.5em;
     text-align: center;
 }
+
+.tile.working {
+    background-color: #bffffb;
+}
+
+.tile.not_working {
+    background-color: #FBA39D;
```
Comment (kiranjthomas) [inline client/src/App.scss]: Maybe we can start defining colors as variables in another file for ease of use in another pr

**Review by kiranjthomas — APPROVED** [review]
Lgtm

**Comment by kiranjthomas** [comment]
Fyi, for my next change set, I'm planning to dockerized the front and backend so we can easily run the project via docker compose. Hope that works for you! If not, let me know.

**Comment by geramirez** [comment]
@kiranjthomas that sounds great, thanks for dockerizing.

**Pull request merged**